src/commands.rs

```diff
@@ -46,21 +47,32 @@ pub fn spawn_network(chain_type: &ChainTypes, binary_path: &str, policy: Option<
     output
 }
 
+/// Return chain_data or wallet_data PathBuf
+pub fn get_wallet_chain_data(chain_type: &ChainTypes, wallet_chain: &str) -> PathBuf {
+    let mut general_dest = get_home_chain(chain_type);
+
+    match wallet_chain {
+        "wallet" => general_dest.push("wallet_data"),
+        "chain" | _ => general_dest.push("chain_data"),
```

Comment:
If it's "chain" or _, it can be just _
```suggestion
        _ => general_dest.push("chain_data"),
```